src/main/java/com/team1816/season/controlboard/ControlBoard.java

```diff
@@ -27,37 +29,14 @@ private ControlBoard(Controller.Factory controller) {
         // demo mode functionality configuration
         if (controlBoardBridge.isDemoMode()) {
             System.out.println("Using Demo Control Board");
+
             demoModeChooser = new SendableChooser<>();
-            SmartDashboard.putData("Demo mode", demoModeChooser);
+            SmartDashboard.putData("Demo Mode", demoModeChooser);
             for (DemoMode demoMode : DemoMode.values()) {
                 demoModeChooser.addOption(demoMode.name(), demoMode);
             }
             demoModeChooser.setDefaultOption(DemoMode.SLOW.name(), DemoMode.SLOW);
-            NetworkTableInstance
-                .getDefault()
-                .getTable("SmartDashboard")
-                .getSubTable("DemoMode")
-                .addEntryListener(
```

Comment:
the listner is how you should be getting the change rather than the disabled check